### DISCUSSION TITLE
China wallet acceptance address

### DISCUSSION THREAD

**Discussion by tracy32792** [Chia-Network/chia-blockchain#6826] [Q&A]
Click the new address under the receipt address of Chia wallet, which causes the wallet receiving address to change. How can he change it to the original wallet receiving address before？

**Comment by loppefaaret** [comment]
you can't, but that is OK, all the addresses ever created by the wallet, is controlled by the wallet - if you ever receive funds on the first receive address, it will always lead into your wallet

**Comment by mariano54** [comment]
`chia keys show` to get the first one. 
They are all controlled by the same wallet though.
